### ISSUE TITLE
Notebook cells stuck in insert mode

### ISSUE THREAD

**Issue by LordSushiPhoenix** [axelfahy/jupyterlab-vim#3]
After installing on jupyterlab 2.1, the extension doesn't appear to do anything. 

The extensions tab shows "@axlair/jupyterlab_vim" is installed and enabled, but cells are stuck in insert mode and pressing escape exits the cell instead of entering vim mode.

**Comment by jrs65** [comment]
Having the same issue here. Not sure what's causing it at the moment. I'm using jupyterlab 2.1.1 and the latest master of this repo, installed via `jupyter labextension install @axlair/jupyterlab_vim`

**Comment by jrs65** [comment]
I just tried downgrading to `jupyterlab==2.0.2` which didn't help at all.

**Comment by jrs65** [comment]
Okay. Somehow my `/usr/local/share/jupyter/lab/settings/page_config.json` file ended up with the extension disabled:
```
{
    "disabledExtensions": [
        "jupyterlab_vim"
    ]
}
```
although both the Extension Manager, and doing `jupyter labextension list` where both showing the extension as disabled. Once I removed this file it all started working again.

I think this happened because I did disable the original upstream extension before installing this one (and then also uninstalling upstream).

Hope that helps you @LordSushiPhoenix
